Sources/Sentry/Public/SentryUser.h

```diff
@@ -39,7 +39,6 @@ NS_SWIFT_NAME(User)
 - (instancetype)initWithUserId:(NSString *)userId;
 
 - (instancetype)init;
-+ (instancetype)new NS_UNAVAILABLE;
```

Comment:
I don't see a reason why this was unavailable? The standard init was available too after all 🤔